src/pages/FoodItem/QuantityInput/QuantityInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import {
+  PlusButton,
+  MinusButton,
+  PlusIcon,
+  MinusIcon,
+} from "../../../assets/svgs";
+import { QuantityInputStyle } from "./QuantityInput.style";
+import { animated, useSpring } from "react-spring";
+import { OperatorDiv } from "./OperatorDiv.style";
+
+type Props = {
+  quantity: number;
+  onClick: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export default React.memo(function QuantityInput({ quantity, onClick }: Props) {
+  const clickHandler = (action: string): void => {
```

Comment:
create a separate handler for incrementing and decrementing so it's simpler

Reply:
Furthermore, if you only expect the input to be either "increment" or "decrement", you can create use a union type for action instead of string. ("increment" | "decrement") This prevents any other inputs to the function

Reply:
When the quantity is 0, Grab's button says back to menu, maybe we can copy that. Either do that or disable the minus button when it's at 1